samples/peripheral/lpuart/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+set(NRF_SUPPORTED_BOARDS
+	nrf5340pdk_nrf5340_cpunet
+	nrf52dk_nrf52832
+	nrf52840dk_nrf52840
```

Comment:
samples should not use the `NRF_SUPPORTED_BOARDS`.

There can be 3rd party boards in Zephyr where this sample will work, for example the reel board.
Or end-user could have a custom board.

Instead, the `sample.yaml::whitelist` should specify the boards that are verified to work with this sample.

Reply:
removed